src/Microsoft.DocAsCode.Build.Engine/Incrementals/IncrementalBuildContext.cs

```diff
@@ -62,7 +79,7 @@ public IncrementalBuildContext(DocumentBuildParameters parameters, BuildVersionI
         /// <param name="hostService">host service</param>
         /// <param name="file">the model's LocalPathFromRoot</param>
         /// <param name="phase">the buildphase that the model was loaded at</param>
-        internal void ReportModelLoadInfo(HostService hostService, string file, BuildPhase? phase)
+        public void ReportModelLoadInfo(HostService hostService, string file, BuildPhase? phase)
```

Comment:
it is not written as hostservice's extensionmethods now, but i feel like it could be later